### ISSUE TITLE
Ionic Upload (Forbidden Upload 403)

### ISSUE THREAD

**Issue by ptmagalhaes** [driftyco/ionic#9887]
An error occurred uploading the build: Forbidden upload (403)

Forbidden upload (403) (CLI v2.1.17)

Your system information:

Cordova CLI: 6.4.0
Ionic Framework Version: 2.0.0-rc.4
Ionic CLI Version: 2.1.17
Ionic App Lib Version: 2.1.7
Ionic App Scripts Version: 0.0.47
ios-deploy version: 1.9.0
ios-sim version: 5.0.13
OS: OS X El Capitan
Node Version: v7.3.0
Xcode version: Xcode 8.2 Build version 8C23

**Comment by wbhob** [comment]
a) Please follow the issue template, it helps a lot with people diagnosing issues.
b) Please share steps to reproduce, because I am having some trouble following what you're doing.

**Comment by ptmagalhaes** [comment]
I Just type (IONIC UPLOAD) . and i see this error

An error occurred uploading the build: Forbidden upload (403)

Forbidden upload (403) (CLI v2.1.17)

Your system information:

Cordova CLI: 6.4.0
Ionic Framework Version: 2.0.0-rc.4
Ionic CLI Version: 2.1.17
Ionic App Lib Version: 2.1.7
Ionic App Scripts Version: 0.0.47
ios-deploy version: 1.9.0
ios-sim version: 5.0.13
OS: OS X El Capitan
Node Version: v7.3.0
Xcode version: Xcode 8.2 Build version 8C23

**Comment by wbhob** [comment]
That may be an ionic backend issue, but please close this and file that at github.com/driftyco/ionic-cli since this is the repo for the framework, and this is a CLI problem.

**Comment by comfortme** [comment]
try 'ionic login' first

**Comment by wbhob** [comment]
I agree with @comfortme, since `403` is an unauthorized error code. You probably aren't logged in.
